broker-util/oo-admin-chk

```diff
@@ -171,7 +171,7 @@ node_hash.each do |gear_uuid, gear_info|
     if !datastore_has_gear?(gear_uuid)
       puts "#{gear_uuid}...FAIL" if $verbose
       print_message "Gear #{gear_uuid} exists on node #{gear_info[0]} (uid: #{gear_info[1]}) but does not exist in mongo database"
-      print_message "Please see https://access.redhat.com/solutions/1171163 for more information."
+      print_message "To fix this issue, remove the gear by deleting /var/lib/openshift/#{gear_uuid} and removing the user from the node host."
```

Comment:
Why not suggest using the `oo-devel-node destroygear` command instead of manually deleting the directory and user?